### ISSUE TITLE
Loss function malfunctioning in Class6.ipynb

### ISSUE THREAD

**Issue by arjuaman** [arjuaman/Deep-learning-using-pytorch#2]
Issue#1
i.e. It's not converging to minima

**Comment by arjuaman** [comment]
Loss function was just for learning purpose, not so to complicate things.
